doc/installation.rst

```diff
@@ -349,9 +349,14 @@ Additional help
 
 *Asking a quick question or want to chat with the dev team?* Stop by our `Gitter chatroom`_.
 
+.. rst-class:: fasb openteams-icon
+
+*Need personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```

Comment:
```suggestion
*Seeking personalized help from expert Spyder consultants?* Visit `OpenTeams`_.
```

Sorry, this was my mistake—need is repeated from a few lines above, when every other line has its own beginning word. I fixed it in my PR against your branch as well, so you don't need to apply this if you accept that.